src/GovUk.Education.ExploreEducationStatistics.Admin/web.config

```diff
@@ -15,49 +12,8 @@
         <requestLimits maxAllowedContentLength="104857600" />
       </requestFiltering>
     </security>
-
     <applicationInitialization doAppInitAfterRestart="true">
       <add initializationPage="/api/health" />
     </applicationInitialization>
-    <rewrite>
-      <rules>
-        <rule name="Redirect azure to govuk - dev" patternSyntax="Wildcard" stopProcessing="true">
-          <match url="*" />
-          <conditions>
-            <add input="{HTTP_HOST}" pattern="s101d01-as-ees-admin.azurewebsites.net" />
```

Comment:
I was thinking we may want to keep these redirects just for the sake of redirecting to the intended domain but I think it would be hard at this point to end up on the azure provided domain so don't think it's a big deal if we remove them

Reply:
Thanks, I'm not aware of anything ending up on the azure domain and I don't think any of these changes should affect Prod so I think it's safe to make. I think we'll end up spotting anything that's broken pretty quickly and can reinstate these if necessary.